composer.json

```diff
@@ -32,38 +32,41 @@
         }
     ],
     "extra": {
-        "zf": {
+        "laminas": {
             "config-provider": "DoctrineORMModule\\ConfigProvider",
             "module": "DoctrineORMModule"
         }
     },
     "require":           {
-        "php":                               "^7.1",
+        "php":                               "^7.2",
         "doctrine/doctrine-module":          "^2.1.3",
-        "doctrine/orm":                      "^2.6.3",
+        "doctrine/orm":                      "^2.6.4",
         "doctrine/dbal":                     "^2.6.0",
         "symfony/console":                   "^3.3 || ^4.0 || ^5.0",
-        "zendframework/zend-stdlib":         "^3.2.1",
-        "zendframework/zend-hydrator":       "^2.3",
-        "zendframework/zend-mvc":            "^3.1",
-        "zendframework/zend-servicemanager": "^3.3"
+        "laminas/laminas-stdlib":         "^3.2.1",
+        "laminas/laminas-hydrator":       "^2.3",
+        "laminas/laminas-mvc":            "^3.1",
+        "laminas/laminas-servicemanager": "^3.3"
```

Comment:
If this will be pinned to doctrine/doctrine-module 3.x version, then may be not necessary because this dependencies duplicate from doctrine-module?
Also in doctrine-module used laminas/laminas-hydrator "^3.0" versions, so there will be conflicts...

Reply:
https://github.com/doctrine/DoctrineORMModule/pull/603/commits/209426d240f9494d969378346f602325a11be4c0

This commit addresses the duplicate repositories in these two repos.